src/main/java/carRacing/RacingCar.java

```diff
@@ -3,19 +3,33 @@
 public class RacingCar {
 
     public static final int MOVE_CONDITION = 4;
-    private int position;
+    private Position position;
+    private CarName name;
 
-    public int getPosition() {
+    public RacingCar(String name) {
+        this.name = new CarName(name);
+        this.position = new Position();
+    }
```

Comment:
```suggestion
    public Car(String name) {
        this(new Position(), new CarName(name));
    }
    
    public Car(Position position, CarName name) {
        this.position = position;
        this.name = name;
    }
```

필드 값을 초기화하는 주 생성자를 추가하고, 부 생성자가 주 생성자를 호출하는 방식으로 구현하는 방식으로 접근해보면 어떨까?
이와 관련해서는 2차 강의 내용 중 참고